.github/workflows/ci-tests.yml

```diff
@@ -42,6 +42,15 @@ jobs:
             source presto-product-tests/conf/product-tests-${{ matrix.config }}.sh &&
               presto-hive-hadoop2/bin/run_hive_s3_tests.sh
           fi
+      - name: Run Azure Wasb Tests
+        env:
+            WASB_CONTAINER: "${WASB_TESTS_WASB_CONTAINER}"
```

Comment:
how do you provide these in github actions? Is this test actually executed in github actions?

Reply:
tbd